backend/spotihue/spotihue.py

```diff
@@ -57,18 +72,23 @@ def _initialize_spotify(
             client_id (str): Spotify client ID.
             client_secret (str): Spotify client secret.
             redirect_uri (str): Spotify redirect URI.
+            redis_client (redis.Redis object): Redis client for Spotify auth token caching.
```

Comment:
I'm still thinking we have the ability to initialize the class without the redis client and add it as a method argument where needed.

Reply:
I like the idea of using redis as our Spotify Oauth cache backend, but of course we don't strictly need to. Do you mean get rid of its usage in `_init_spotify`, or something else (`_init_spotify` is the only place where the client is used)?